pim-tool-back/src/main/java/vn/elca/training/model/dto/GroupDto.java

```diff
@@ -0,0 +1,18 @@
+package vn.elca.training.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+/**
+ * DTO for {@link vn.elca.training.model.entity.Group}
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class GroupDto implements Serializable {
```

Comment:
[N] When implementing Serializable, also generate a random number for constant `serialVersionUID`.